### PR TITLE
Peer feedback

### PR DESCRIPTION
**Pull request by DuaneDave**
In this PR the todolist project was reset up to receive feedback from a colleague

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ class TodoItem {
     this.completed = completed;
     this.id = id;
   }
-}
+};
 
 class TodoArray {
   constructor() {
```
Comment (soulemanou-software) [inline index.js:10]: Changes can be made but it seems, I can't highlight them to you here in your PR,

**Review by soulemanou-software — COMMENTED** [review]
## Good Job @DuaneDave 
quality-quality-work

- Great work you made the code look simple to understand and easier to debug.
- You respected GitHub flow
- Your codes are dry,  repetition is not available
- The repo has been documented professionally

Other suggestions regarding this PR are added in the comment section and do not fail to reach out if there are any suggestions you are not clear on 😄.

👏 Cheers and happy coding

File: index.js (hunk 1 above)
Comment (DuaneDave) [reply index.js:10]: Thank you for the comment 🙏

Comment (DuaneDave) [reply index.js:10]: Can i go ahead to merge this PR

**Review by soulemanou-software — APPROVED** [review]
## @DuaneDave Excellent job
##
200w
### Nothing else to say than ...
git-merge

**Pull request merged**